test/sqlite.spec.ts

```diff
@@ -9,7 +9,7 @@ describe('sqlite', () => {
 
   before(() => {
     database = knex({
-      client: 'sqlite3',
+      client: 'better-sqlite3',
```

Comment:
we need both tests. can you execute this whole suite in a cycle with two different params?

Reply:
Fixed.